sdk/core/System.ClientModel/src/Pipeline/ClientPipeline.cs

```diff
@@ -144,6 +154,10 @@ public static ClientPipeline Create(
 
         int perTryIndex = index;
 
+        // Add logging policy just before the transport.
+
+        policies[index++] = options.LoggingPolicy ?? new ClientLoggingPolicy(options.LoggingOptions);
+
         // Before transport policies come before the transport.
```

Comment:
I read this comment in his voice:
![image](https://github.com/user-attachments/assets/a5077fd8-86c1-49e5-b3c6-cab8747d528c)